app/controllers/organizations/new.js

```diff
@@ -47,43 +51,161 @@ export default class OrganizationsNewController extends Controller {
+      return this.store.createRecord('central-worship-service');
+    } else if (classificationCodeId === CLASSIFICATION.WORSHIP_SERVICE.id) {
+      return this.store.createRecord('worship-service');
+    } else if (
+      [
+        CLASSIFICATION.MUNICIPALITY.id,
+        CLASSIFICATION.PROVINCE.id,
+        CLASSIFICATION.OCMW.id,
+        CLASSIFICATION.DISTRICT.id,
+        CLASSIFICATION.AGB.id,
+        CLASSIFICATION.APB.id,
+        CLASSIFICATION.PROJECTVERENIGING.id,
+        CLASSIFICATION.DIENSTVERLENENDE_VERENIGING.id,
+        CLASSIFICATION.OPDRACHTHOUDENDE_VERENIGING.id,
+        CLASSIFICATION.OPDRACHTHOUDENDE_VERENIGING_MET_PRIVATE_DEELNAME.id,
+        CLASSIFICATION.POLICE_ZONE.id,
+        CLASSIFICATION.ASSISTANCE_ZONE.id,
+        CLASSIFICATION.WELZIJNSVERENIGING.id,
+        CLASSIFICATION.AUTONOME_VERZORGINGSINSTELLING.id,
+        CLASSIFICATION.PEVA_MUNICIPALITY.id,
+        CLASSIFICATION.PEVA_PROVINCE.id,
+      ].includes(classificationCodeId)
+    ) {
+      return this.store.createRecord('administrative-unit');
+    } else {
+      return this.store.createRecord('organization');
+    }
+  }
+
+  /**
+   * Copy the properties of the {@link currentOrganizationModel} to the provided
+   * model instance. Any relations the {@link currentOrganizationModel} has with
+   * other organizations are *not* copied. These relations are dependent upon
+   * the exact classification of the organization and should be explicitly set
+   * by the user.
+   *
+   * @param {OrganizationModel} newOrganizationModel - the model instance to
+   * which the properties must be copied
+   */
+  #copyPropertiesToModel(newOrganizationModel) {
+    newOrganizationModel.name = this.currentOrganizationModel.name;
+    newOrganizationModel.legalName = this.currentOrganizationModel.legalName;
+    newOrganizationModel.alternativeName =
+      this.currentOrganizationModel.alternativeName;
+    newOrganizationModel.organizationStatus =
+      this.currentOrganizationModel.organizationStatus;
+
+    if (newOrganizationModel.isWorshipAdministrativeUnit) {
+      newOrganizationModel.recognizedWorshipType =
+        this.currentOrganizationModel.recognizedWorshipType;
+    }
+
+    if (this.currentOrganizationModel.scope) {
+      newOrganizationModel.scope = this.currentOrganizationModel.scope;
+      if (newOrganizationModel.scope.locatedWithin) {
+        newOrganizationModel.scope.locatedWithin =
+          this.currentOrganizationModel.scope.locatedWithin;
+      }
+    }
+
+    // Only IGSs have an, optional, expected end data and/or goal. Only copy
+    // these values when new organization is also an IGS.
+    if (newOrganizationModel.isIgs) {
+      newOrganizationModel.expectedEndDate =
+        this.currentOrganizationModel.expectedEndDate;
+      if (this.currentOrganizationModel.purpose?.length) {
+        newOrganizationModel.purpose = this.currentOrganizationModel.purpose;
+      }
```

Comment:
**Question:** Is it normal to have less properties copied in the new function than in the original one (eg. `wasFoundedByOrganizations`, `isAssociatedWith`...)? https://github.com/lblod/frontend-organization-portal/pull/611/files#diff-85ba2ee0843881ee36cae41399c083e5cd4141e9ad065e1cc2ca28c30edce447L205-L246

Reply:
I found the answer in the PR description, you can forget that comment ;)